### PR TITLE
[refactor]: rename permission validator into validator

### PR DESCRIPTION
**Pull request by mversic**
## Description

It is now incorrect to refer to the validator as permission validator. 
Permission validation is a subset of runtime validation

### Linked issue

Closes #{issue_number} 

### Checklist

- [ ] I've read `CONTRIBUTING.md`
- [ ] I've used the standard signed-off commit format (or will squash just before merging)
- [ ] All applicable CI checks pass (or I promised to make them pass later)
- [ ] (optional) I've written unit tests for the code changes
- [ ] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

**Review by appetrosyan — DISMISSED** [review]

File: data_model/src/permissions.rs
Comment (pesterev) [inline data_model/src/permissions.rs:1]: Why rename it to plural? I thought we always named modules in the singular.

File: schema/gen/src/lib.rs [hunk 1]
```diff
@@ -313,7 +313,7 @@ types!(
     PermissionTokenDefinition,
     PermissionTokenEvent,
     PermissionTokenId,
-    PermissionValidatorEvent,
+    ValidatorEvent,
```
Comment (pesterev) [inline schema/gen/src/lib.rs:316]: Let's keep it in alphabet order

File: data_model/src/permissions.rs
Comment (mversic) [reply data_model/src/permissions.rs:1]: ok

**Review by pesterev — APPROVED** [review]
LGTM

**Review by appetrosyan — APPROVED** [review]

**Pull request merged**